lib/player.rb

```diff
@@ -0,0 +1,8 @@
+class Player
+  attr_accessor :name, :sign
```

Comment:
[Please fix] This is a perfect example of using `attr_reader` because you are only setting the instance variable when you create the object, then you never need to change the variable again, the reason why you just need the read attribute.